### PR TITLE
68 organization query and mutation

### PR DESCRIPTION
**Pull request by ignasposka**
Added validation solution as well.

### CODE REVIEW

File: src/sql-queries/organization.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  id: number;
+  name: String;
+  country: String;
+  city: String;
+  streetAddress: String;
+  phone: String;
+}
+
+export const getOrganizationQuery = (id: number): QueryConfig =>
+  select().from('organization').where({ id }).toParams();
```
Comment (JBK91) [inline src/sql-queries/organization.ts:23]: Shouldn't select queries filter rows where delete_time IS NOT NULL?

Comment (AzisK) [inline src/sql-queries/organization.ts:23]: Should we go with this ORM approach? I believe I would prefer pure SQL as we did before

Comment (ignasposka) [reply src/sql-queries/organization.ts:23]: I guess you're right.

Comment (ignasposka) [reply src/sql-queries/organization.ts:23]: This is not ORM, just sql helper. Partial update gets very messy with pure sql (discussed with @psaulenas).

**Review by AzisK — APPROVED** [review]

**Pull request merged**